src/views/network/BoardListing.vue

```diff
@@ -115,10 +114,11 @@ export default {
         { value: 'other', name: 'Other' }
       ],
       isLoading: false,
+      source: null,
```

Comment:
I don't think the cancel token is working correctly, and adding this line did not fix it

Reply:
Does this info help you? https://github.com/axios/axios#cancellation

Reply:
Ah it should be fixed with the newest commit